CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## [0.20.1](https://github.com/dashevo/dapi-client/compare/v0.20.0...v0.20.1) (2021-05-26)
+
+
+### Bug Fixes
+
+* * crash when connecting to an older version of dapi ([#252](https://github.com/dashevo/dapi-client/issues/252))
```

Comment:
Two stars. Also, I guess we have two fixes: one for platform endpoints and another one for getTransaction what we forgot to do initially. I such cases we usually list several PRs for one item.